ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/functions/[slug]/logs/NewReplayModal.tsx

```diff
@@ -109,14 +109,15 @@ export default function NewReplayModal({ functionSlug, isOpen, onClose }: NewRep
     FunctionRunStatus.Failed,
   ]);
   const environment = useEnvironment();
-  const { data, isLoading, error } = useGraphQLQuery({
+  const { data, isLoading, error } = useSkippableGraphQLQuery({
     query: GetFunctionEndedRunsCountDocument,
     variables: {
       environmentID: environment.id,
       functionSlug,
       timeRangeStart: timeRange?.start ? timeRange.start.toISOString() : '',
       timeRangeEnd: timeRange?.end ? timeRange.end.toISOString() : '',
     },
+    skip: !timeRange,
```

Comment:
Do we need to check both `start` and `end` exist too?

Reply:
`start` and `end` are required properties so `!timeRange` is fine. But I changed the ternary to better reflect that